.docker/mariadb/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mariadb
+
+RUN apt-get update && \
+    apt-get install -y tzdata mycli && \
+    apt-get clean && \
+    rm -fr /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN cp /usr/share/zoneinfo/America/Sao_Paulo /etc/localtime && \
+    echo "America/Sao_Paulo" > /etc/timezone
```

Comment:
We can remove this and only use the environment TZ:

```bash
~$ docker run -it -e TZ=GMT mariadb bash
root@be6576b7807c:/# date
Mon Jan  8 13:42:18 GMT 2024
root@be6576b7807c:/# 
exit
~$ docker run -it -e TZ=America/Sao_Paulo mariadb bash
root@2d3b14a0553e:/# date
Mon Jan  8 10:42:30 -03 2024
~$ docker run -it -e TZ mariadb bash
root@9fe717c604ca:/# date
Mon Jan  8 13:43:40 UTC 2024
```

If we wish to change, we can set a default value at docker-compose.yml:
```yaml
- TZ=${TZ:-America/Sao_Paulo}
```